website/docs/r/management_group_template_deployment.html.markdown

```diff
@@ -0,0 +1,137 @@
+        "policyDefinitionId": "[parameters('policyDefinitionID')]"
+      }
+    }
+  ]
+}
+TEMPLATE
+
+  parameters_content = <<PARAMS
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "policyDefinitionID": {
+      "value": "/providers/Microsoft.Authorization/policyDefinitions/0a914e76-4921-4c19-b460-a2d36003525a"
+    }
+  }
+}
+PARAMS
+}
+
+// NOTE: whilst we show an inline template here, we recommend
+// sourcing this from a file for readability/editor support
+
+```
+
+```hcl
+data "azurerm_template_spec_version" "example" {
+  name                = "myTemplateForManagementGroup"
+  resource_group_name = "myResourceGroup"
+  version             = "v1.0.9"
+}
+
+resource "azurerm_management_group_template_deployment" "example" {
+  name                  = "example"
+  location              = "West Europe"
+  management_group_name = "example"
+
+  template_spec_version_id = data.azurerm_template_spec_version.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region where the Template should exist. Changing this forces a new Template to be created.
+
+* `management_group_name` - (Required) The Name of the Management Group to apply the Deployment Template to.
+
+* `name` - (Required) The name which should be used for this Template Deployment. Changing this forces a new Template Deployment to be created.
+
+---
+
+* `debug_level` - (Optional) The Debug Level which should be used for this Resource Group Template Deployment. Possible values are `none`, `requestContent`, `responseContent` and `requestContent, responseContent`. 
+
+* `parameters_content` - (Optional) The contents of the ARM Template parameters file - containing a JSON list of parameters.
+
+* `template_content` - (Optional) The contents of the ARM Template which should be deployed into this Resource Group. Cannot be specified with `template_spec_version_id`. 
+
+* `template_spec_version_id` - (Optional) The ID of the Template Spec Version to deploy. Cannot be specified with `template_content`.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Template.
+
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Management Group Template Deployment.
+
+* `output_content` - The JSON Content of the Outputs of the ARM Template Deployment.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 3 hours) Used when creating the Template.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Template.
+* `update` - (Defaults to 3 hours) Used when updating the Template.
+* `delete` - (Defaults to 3 hours) Used when deleting the Template.
```

Comment:
seems long for management group level resources? (also should this end "[Template] Deployment"?)

Reply:
The template spec items can have chained templates, so could possibly (likely?) include templates that deploy into items directly created at the management group.